swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0.0):
+        if not id:
+            id = uuid.uuid4().int
+        self.id = id
+        self.condition = condition
+
+    def get_category(self):
+        if isinstance(self, Item):
```

Comment:
This check is not necessary since only Item instances can call this method. You can just return "Item".